code/game/objects/items/melee/misc.dm

```diff
@@ -505,3 +505,82 @@
 	held_sausage.name = "[target.name]-roasted [held_sausage.name]"
 	held_sausage.desc = "[held_sausage.desc] It has been cooked to perfection on \a [target]."
 	update_icon()
+
+/obj/item/twohanded/required/electrostaff
+	icon = 'icons/obj/estaff.dmi'
+	icon_state = "electrostaff_0"
+	item_state = "electrostaff_0"
+	lefthand_file = 'icons/mob/inhands/weapons/estaff_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/estaff_righthand.dmi'
+	name = "riot suppression electrostaff"
+	desc = "A large quarterstaff, with massive silver electrodes mounted at the end."
+	force = 1
+	damtype = "fire"
+	w_class = WEIGHT_CLASS_HUGE
+	slot_flags = ITEM_SLOT_BACK
+	sharpness = FALSE
+	force_unwielded = 1
+	force_wielded = 1
+	throwforce = 1
+	throw_speed = 1
+	light_range = 7
+	light_power = 1
+	block_chance = 30
+	light_color = LIGHT_COLOR_YELLOW
+	materials = list(MAT_METAL=1000)
+	hitsound = 'sound/weapons/staff.ogg'
+	attack_verb = list("suppresed", "struck", "beaten", "thwacked", "pulped", "shocked")
+	total_mass = 5
+	var/power = "suppresive"
+	var/lethal = FALSE
+	var/stun_stam_cost_coeff = 1.25
+	var/hardstun_ds = 1
+	var/softstun_ds = 2
+	var/stam_dmg = 45
```

Comment:
Not even a taser bolt does that much... 
Perhaps 30? So its in line with other stun based weaponds - Stun-Gun